TripKitUI/core/TransportKitUI/SGTripSegmentsView.m

```diff
@@ -104,13 +104,17 @@ - (void)configureForSegments:(NSArray<id<STKTripSegmentDisplayable>> *)segments
 		}
 		
 		// for the coloured strip
-		UIColor *color = (allowSubtitles && [segment respondsToSelector:@selector(tripSegmentModeColor)])
-      ? [segment tripSegmentModeColor]
-      : nil;
+//    UIColor *color = (allowSubtitles && [segment respondsToSelector:@selector(tripSegmentModeColor)])
+//      ? [segment tripSegmentModeColor]
+//      : nil;
+    
+    UIColor *color = (self.colorCodingTransit && [segment respondsToSelector:@selector(tripSegmentModeColor)])
```

Comment:
I think this break the Chicago case where a line has a colour but no title and the `SGTripSegmentsView` would previously show the colour for it next to the icon. In this case `self.colorCodingTransit` is false but `allowSubtitles` is true. With this change `color` would be set to `nil` and we'd never show it.